resources/views/components/widget.blade.php

```diff
@@ -164,7 +164,7 @@ class="cx-w-full cx-text-base cx-font-normal cx-leading-6 cx-transition-colors c
                             <details class="cx-group cx-w-full cx-group cx-px-4 {{ $this->css('cxFaqText') }}">
                                 <summary class="cx-w-full cx-flex cx-gap-2 cx-cursor-pointer cx-list-none cx-items-center cx-py-4 cx-text-sm cx-font-semibold">
                                     <div class="cx-w-8 cx-flex cx-items-center {{ $this->css('cxFaqTitleArea') }}">
-                                        {!! $faq->getFirstMedia('Icon')->class('cx-w-full cx-object-contain ' . $this->css('cxFaqTitleIcon')) !!}
+                                        {!! $faq->load('media')->getFirstMedia('Icon')->class('cx-w-full cx-object-contain ' . $this->css('cxFaqTitleIcon')) !!}
```

Comment:
Using `->loadMissing('media')` is a more optimized call, as it will not make any additional database calls if the relationship has already been loaded. [Documentation](https://laravel.com/docs/10.x/eloquent-collections#method-loadMissing).

```suggestion
                                        {!! $faq->loadMissing('media')->getFirstMedia('Icon')->class('cx-w-full cx-object-contain ' . $this->css('cxFaqTitleIcon')) !!}
```

Reply:
Thanks!